dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -10,6 +10,7 @@
 - hide_source ||= false
 - use_phaser ||= false
 - use_makerlab ||= false
+- app_options[:locale] = js_locale
```

Comment:
Why did we miss locale (and possibly disableSocialShare) initially? I'm a little scared of just depending on our test coverage to catch mistakes here.